docs/api-v1.md

```diff
@@ -0,0 +1,280 @@
+# TabPy API v1
+
+<!-- markdownlint-disable MD004 -->
+
+<!-- toc -->
+
+- [Authentication](#authentication)
+- [http:get:: /status](#httpget-status)
```

Comment:
We don't document /info? It's a part of the API.

Reply:
Info is outside of any API version and is documented at server-rest.md